ext/drsyms/elfutils/elf_memory.c.patch

```diff
@@ -1,11 +0,0 @@
-diff --git a/libelf/elf_memory.c b/libelf/elf_memory.c
```

Comment:
>  As this release also applied the elf_memory.c fix, we drop it here:
>
> cc44ac67 libelf: Treat elf_memory as if using ELF_C_READ_MMAP

Version 0.192, despite our protests, reverts this in 81981fdfd8f776508579da60d29e068359c61e6a and breaks our use case: so we will need this patch again.  Although we are now using a local fork of the elfutils repo (because of #7253) so we could patch there.

Reply:
That's unfortunate, but in the end it does not matter much, as we anyways need patches. I actually proposed this MR due to #7253. I'm building DR in a Yocto recipe where I manually pull together the third party components. There it helps a lot to download release versions which you can get from mirrors (like the [kernel CDN](http://mirrors.kernel.org/sourceware/elfutils/0.192/)) and ideally the sources also only have to be downloaded once.

Reply:
Do you plan to replace the clone we made to solve #7253?

Reply:
I just replaced it locally to fetch the external parts (`third_party`) with bitbake. I don't plan to replace it here. But to make it easier to use external sources, sticking to releases helps. We could replace it with an `AddExternalProject` cmake call, but this also has its down sides. For now, let's just stick to the submodule approach in combination with the local fork.